src/zope/app/locales/extract.py

```diff
@@ -231,6 +236,7 @@ def _getProductVersion(self):
         return "Unknown"
 
     def write(self):
+        pot_header = self._pot_header
         with open(self._output_filename, 'wb') as file:
             formatted_header = pot_header % {
```

Comment:
```suggestion
            formatted_header = self._pot_header % {
```
Is there something wrong with directly using the attribute here? As this seems to be the only usage of `pot_header` it might be possible to omit the definition at the top of the file.

Reply:
Right, I blindly copied the code from `z3c.recipe.i18n` here.